common/lib/capa/capa/tests/response_xml_factory.py

```diff
@@ -193,6 +230,12 @@ def create_response_element(self, **kwargs):
             responseparam_element = etree.SubElement(response_element, 'responseparam')
             responseparam_element.set('type', 'tolerance')
             responseparam_element.set('default', str(tolerance))
+            if partial_range is not None and 'close' in credit_type:
+                responseparam_element.set('partial_range', str(partial_range))
+
+        if partial_answers is not None and 'list' in credit_type:
+            responseparam_element = etree.SubElement(response_element, 'responseparam')
```

Comment:
@sarina @Colin-Fredericks Indeed that file has these violations already. See report on your last build [here](https://build.testeng.edx.org/job/edx-platform-test-subset/131503/artifact/reports/common/pylint.report/*view*/). The intention of using diff-quality is so that, at the least, we don't continue a bad pattern. So, you ought to fix it on the new lines.

That said, this particular error can give false positives, as pointed out [here](http://pylint-messages.wikidot.com/messages:e1101). I'm only glancing at this code, but if this is a dynamically created member, then perhaps this applies to you.